ansible/roles/openmpi/tasks/hostfile.yml

```diff
@@ -0,0 +1,6 @@
+---
```

Comment:
This native MPI installation is only used for benchmarking our experiments, so isn't useful in "normal" faasm day-to-day work. The `openmpi` role already exists in the `faasm-experiments` repo (https://github.com/faasm/faasm-experiments/tree/master/ansible/roles/openmpi) so we should use that.

The general rule should be that all experimental code and benchmarks should live in `faasm-experiments`, and this repo should just contain the Faasm system.

It looks like an old `openmpi.yml` playbook was left in this repo by accident, so we should delete that.